llvm/test/CodeGen/AMDGPU/frame-index.mir

```diff
@@ -211,3 +211,327 @@ body:             |
     renamable $vgpr0 = V_ADD_CO_U32_e32 %stack.0, killed $vgpr0, implicit-def dead $vcc, implicit $exec
 ...
 
+---
```

Comment:
In a pre-commit, can you generate GFX8 and GFX9 checks in this test

Reply:
sure.

Reply:
addressed.